lib/widgets/inputs/custom_date_time_picker.dart

```diff
@@ -51,41 +48,38 @@ class _CustomDateAndTimePickerState extends State<CustomDateAndTimePicker> {
           data: ThemeData.light().copyWith(
             primaryColor: Colors.black,
             buttonTheme: ButtonThemeData(textTheme: ButtonTextTheme.accent),
-            dialogBackgroundColor: Colors.white, // Weißer Hintergrund
+            dialogBackgroundColor: Colors.white,
           ),
           child: child!,
         );
       },
     );
 
     if (pickedDate != null) {
-      // Zeit-Auswahl
       TimeOfDay? pickedTime = await showTimePicker(
         context: context,
         initialTime: TimeOfDay.fromDateTime(_selectedDateTime),
       );
 
       if (pickedTime != null) {
         setState(() {
-          // Kombiniere Datum und Uhrzeit
           _selectedDateTime = DateTime(
             pickedDate.year,
             pickedDate.month,
             pickedDate.day,
             pickedTime.hour,
             pickedTime.minute,
           );
-          _dateController.text = DateFormat('yyyy-MM-dd HH:mm').format(_selectedDateTime);
+          _dateController.text =
+              DateFormat('yyyy-MM-dd HH:mm').format(_selectedDateTime);
```

Comment:
Wir müssen uns noch darauf einigen wie wir Datum und darstellen...
Ist gerade immer anders in Event Detail View, Input Fields und Date und time picker
-> Zeit in 24h oder 12h (mit am/pm)
-> Datum als "YYYY-MM-DD" oder "DD.MM.YYYY"